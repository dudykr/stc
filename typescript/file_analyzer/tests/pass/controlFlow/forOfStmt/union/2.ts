var union: string | string[];
for (const v of union) {
    v
}

export { }