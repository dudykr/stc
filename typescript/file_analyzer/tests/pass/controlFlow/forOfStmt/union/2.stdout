warning: Type
 --> $DIR/tests/pass/controlFlow/forOfStmt/union/2.ts:2:17
  |
2 | for (const v of union) {
  |                 ^^^^^
  |
  = note: string | string[]

warning: Type
 --> $DIR/tests/pass/controlFlow/forOfStmt/union/2.ts:3:5
  |
3 |     v
  |     ^
  |
  = note: string

